2016-02-25 09:50:25,684 - [INFO] - from play in pool-4-thread-2 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2016-02-25 09:52:13,220 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 09:52:15,245 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2016-02-25 09:52:19,377 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 09:52:19,572 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2016-02-25 09:55:54,530 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 09:55:54,724 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2016-02-25 09:57:38,521 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 09:57:38,701 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2016-02-25 09:57:57,982 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 09:57:58,142 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2016-02-25 09:58:10,726 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 09:58:10,902 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2016-02-25 10:00:08,657 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:00:08,830 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2016-02-25 10:00:46,196 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:00:46,388 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2016-02-25 10:03:31,289 - [INFO] - from play in play-internal-execution-context-4 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:03:31,467 - [INFO] - from play in play-internal-execution-context-4 
Application started (Dev)

2016-02-25 10:10:16,221 - [ERROR] - from application in New I/O worker #4 


! @6p8bm5gpf - Internal server error, for (GET) [/] ->

play.PlayExceptions$RoutesCompilationException: Compilation error[Missing parameter in call definition: path]
	at play.PlaySourceGenerators$$anonfun$2.apply(PlaySourceGenerators.scala:21) ~[na:na]
	at play.PlaySourceGenerators$$anonfun$2.apply(PlaySourceGenerators.scala:9) ~[na:na]
	at play.Settings$$anonfun$defaultSettings$18.apply(PlaySettings.scala:105) ~[na:na]
	at play.Settings$$anonfun$defaultSettings$18.apply(PlaySettings.scala:104) ~[na:na]
	at scala.Function6$$anonfun$tupled$1.apply(Function6.scala:35) ~[scala-library.jar:na]
	at scala.Function6$$anonfun$tupled$1.apply(Function6.scala:34) ~[scala-library.jar:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[scala-library.jar:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_60]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2016-02-25 10:11:19,381 - [ERROR] - from application in New I/O worker #4 


! @6p8bm96dl - Internal server error, for (GET) [/] ->

play.PlayExceptions$RoutesCompilationException: Compilation error[Missing parameter in call definition: path]
	at play.PlaySourceGenerators$$anonfun$2.apply(PlaySourceGenerators.scala:21) ~[na:na]
	at play.PlaySourceGenerators$$anonfun$2.apply(PlaySourceGenerators.scala:9) ~[na:na]
	at play.Settings$$anonfun$defaultSettings$18.apply(PlaySettings.scala:105) ~[na:na]
	at play.Settings$$anonfun$defaultSettings$18.apply(PlaySettings.scala:104) ~[na:na]
	at scala.Function6$$anonfun$tupled$1.apply(Function6.scala:35) ~[scala-library.jar:na]
	at scala.Function6$$anonfun$tupled$1.apply(Function6.scala:34) ~[scala-library.jar:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[scala-library.jar:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_60]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2016-02-25 10:13:38,300 - [INFO] - from play in play-internal-execution-context-4 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:13:38,462 - [INFO] - from play in play-internal-execution-context-4 
Application started (Dev)

2016-02-25 10:13:39,737 - [INFO] - from play in play-internal-execution-context-4 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:13:39,917 - [INFO] - from play in play-internal-execution-context-4 
Application started (Dev)

2016-02-25 10:14:13,480 - [INFO] - from play in play-internal-execution-context-4 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:14:13,693 - [INFO] - from play in play-internal-execution-context-4 
Application started (Dev)

2016-02-25 10:15:24,512 - [INFO] - from play in play-internal-execution-context-5 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:15:24,670 - [INFO] - from play in play-internal-execution-context-5 
Application started (Dev)

2016-02-25 10:16:31,603 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:16:31,753 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2016-02-25 10:23:33,795 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:23:33,954 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2016-02-25 10:23:34,858 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:23:35,023 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2016-02-25 10:24:02,802 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:24:02,967 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2016-02-25 10:24:48,577 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:24:48,742 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2016-02-25 10:46:45,504 - [INFO] - from play in play-internal-execution-context-17 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:46:45,681 - [INFO] - from play in play-internal-execution-context-17 
Application started (Dev)

2016-02-25 10:47:38,923 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-25 10:47:39,067 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

